package.json

```diff
@@ -25,14 +26,24 @@
     "@babel/preset-react": "^7.0.0",
     "babel-loader": "^8.0.6",
     "css-loader": "^3.2.0",
+    "eslint": "6.1.0",
+    "eslint-config-airbnb": "18.0.1",
+    "eslint-config-prettier": "^6.4.0",
+    "eslint-plugin-import": "2.18.2",
+    "eslint-plugin-jsx-a11y": "6.2.3",
+    "eslint-plugin-react": "7.14.3",
+    "eslint-plugin-react-hooks": "1.7.0",
+    "react": "^16.8.0",
     "react-dom": "^16.9.0",
     "style-loader": "^1.0.0",
     "webpack": "^4.39.3",
     "webpack-cli": "^3.3.7",
-    "webpack-dev-server": "^3.8.0",
-    "react": "^16.8.0"
+    "webpack-dev-server": "^3.8.0"
   },
   "peerDependencies": {
     "react": "^16.8.0"
+  },
+  "dependencies": {
```

Comment:
Can we move this to dev dependencies please? Don't think we need it as a dependency once the bundle is formed. That way, the library size will remain small :)